src/index.js

```diff
@@ -33,17 +33,19 @@ const messages = {
     fr: { ...messages_fr, ...login_fr, ...top_bar_fr },
 };
 
-const language = navigator.language.split(/[-_]/)[0]; // language without region code
-
 const basename = new URL(document.querySelector('base').href).pathname;
 
-ReactDOM.render(
-    <IntlProvider locale={language} messages={messages[language]}>
-        <Provider store={store}>
-            <BrowserRouter basename={basename}>
-                <App />
-            </BrowserRouter>
-        </Provider>
-    </IntlProvider>,
-    document.getElementById('root')
-);
+const ApplicationWrapper = () => {
+    const [language, setLanguage] = useState();
```

Comment:
what is the intial value of this state !  you should initialize it with SYSTEM ?

Reply:
there should not be a state here since the language is stored in redux